src/auth/CMakeLists.txt

```diff
@@ -28,3 +28,5 @@ if(WITH_SEASTAR)
   target_include_directories(crimson-auth PRIVATE
     $<TARGET_PROPERTY:Seastar::seastar,INTERFACE_INCLUDE_DIRECTORIES>)
 endif()
+
+target_include_directories(common-auth-objs PRIVATE ${OPENSSL_INCLUDE_DIR})
```

Comment:
@petrutlucian94 
could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#commit-title . in this case, it would be "cmake: ".

Reply:
Sure. Is there a list of subcomponents that I can choose from? If a certain commit touches more than one subcomponent (e.g. five of them) but simply renames a constant, do we still need a prefix?

Reply:
> Is there a list of subcomponents that I can choose from? 

no, but i think a rule of thumb is to check the output of "git log <file-you-are-changing>". 

> touches more than one subcomponent (e.g. five of them) but simply renames a constant, do we still need a prefix?

yes, for instance, if you are changing a file under "src/common", and another file under "src/rgw", i'd suggest use "common,rgw: " as the prefix.

Reply:
``cmake`` was already part of the commit message, I guess I'll go with "win32, cmake: handle Windows targets" to avoid being redundant (well, it still sort of is).

Reply:
no, please move "win32" out of the prefix. please note, "win32" is not a subcomponent of ceph. it's hard to map it to a certain directory or file in the source tree.

Reply:
Can that still be part of the commit title? As I was mentioning, it simplifies scoping the commits and also makes them easier to find.